### PR TITLE
fix: Users can not join expired beacons.

### PR DESCRIPTION
**Pull request by Aadeesh11**
Resolves: #73

### CODE REVIEW

File: graphql/resolvers.js [hunk 1]
```diff
@@ -117,6 +117,7 @@ const resolvers = {
             const beacon = await Beacon.findOne({ shortcode });
 
             if (!beacon) return new UserInputError("No beacon exists with that shortcode.");
+            if(beacon.expiresAt < Date()) return new Error("Beacon has expired");
```
Comment (aadibajpai) [inline graphql/resolvers.js:120]:
```suggestion
            if (beacon.expiresAt < Date()) return new Error("Beacon has expired");
```

**Comment by aadibajpai** [comment]
have you tested this locally?

**Comment by Aadeesh11** [comment]
> have you tested this locally?

No I could not, since I have not set up the backend locally.

**Comment by aadibajpai** [comment]
> > have you tested this locally?
> 
> No I could not, since I have not set up the backend locally.

it would be preferable to test it out, even though it is a trivial change it sets you up well to work on this further

**Comment by Aadeesh11** [comment]
> > > have you tested this locally?
> > 
> > 
> > No I could not, since I have not set up the backend locally.
> 
> it would be preferable to test it out, even though it is a trivial change it sets you up well to work on this further

would try to set it up👍

**Comment by Aadeesh11** [comment]
@aadibajpai I have tested the changes locally, it works. This pr is ready to be merged.

**Pull request merged**